src/main/java/S01/Calculator.java

```diff
@@ -0,0 +1,27 @@
+package S01;
+
+import java.util.HashMap;
+import java.util.function.BiFunction;
+
+public class Calculator {
+    /*
+        Calculator Class
+        ver 1.0
+        2020.06.22
+     */
+    private final HashMap<String, BiFunction<Double,Double,Double>> calculatorHashMap; // 계산
```

Comment:
```suggestion
    private final Map<String, BiFunction<Double,Double,Double>> calculatorHashMap; // 계산
```

위와 같이 사용하는 것이 어떨까요?!
이렇게 했을 때, 어떤 이점이 있을까요?

현재 변수명을 자료형 이름을 사용했는데
만약 구현체가 변경되면 해당 객체를 사용하는 모든 곳에 이름이 변경되어야 하는 문제가 있지 않을까요?

Reply:
그렇군요!! 좀 더 유지보수하기가 편해질것 같네요!!ㅎㅎ